torchmultimodal/modules/encoders/clip_text_encoder.py

```diff
@@ -91,8 +91,9 @@ def build_attention_mask(self) -> torch.Tensor:
 
     def forward(self, text: torch.Tensor) -> torch.Tensor:
         embeddings = self.encoder(text, attn_mask=self.build_attention_mask())
+        if not isinstance(embeddings, torch.Tensor):
```

Comment:
Why is this needed?

Reply:
aten::permute (used by torchscript on the next line) only takes tensors as input, but the encoder output is annotated as either a tensor or a list of tensors